javascript/modules/client/src/internal/client-addressList/encoding.ts

```diff
@@ -0,0 +1,117 @@
+import { InvalidAddressError } from "@aragon/sdk-common";
+import { ContextPlugin } from "../../context-plugin";
+import { ClientCore } from "../core";
+import {
+  encodeAddMembersAction,
+  encodeAddressListActionInit,
+  encodeRemoveMembersAction,
+  encodeUpdatePluginSettingsAction,
+} from "../encoding/plugins";
+import { DaoAction, IPluginInstallItem } from "../interfaces/common";
+import {
+  IAddressListPluginInstall,
+  IClientAddressListEncoding,
+  IPluginSettings,
+} from "../interfaces/plugins";
+import { isAddress } from "@ethersproject/address";
+
+const PLUGIN_ID = "0x1234567890123456789012345678901234567890";
```

Comment:
This may be out of the scope, but this PLUGIN_ID could be defined in a central  `constants.ts` file, and imported here. Otherwise, getting visibility of these values will be hard